presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayJoin.java

```diff
@@ -121,6 +124,11 @@ public ArrayJoin()
                 false));
     }
 
+    public static Object createState()
```

Comment:
Why `Object` and not `PageBuilder`?

Reply:
I assume this will be used by generated code; annotate it with `UsedByGeneratedCode`